ocs_ci/ocs/resources/mcg.py

```diff
@@ -928,48 +935,54 @@ def retrieve_noobaa_cli_binary(self):
             AssertionError: In the case CLI binary doesn't exist.
 
         """
-        if not os.path.isfile(constants.NOOBAA_OPERATOR_LOCAL_CLI_PATH):
+        logger.info(
+            f"The MCG CLI binary could not be found in {constants.NOOBAA_OPERATOR_LOCAL_CLI_PATH},"
+            " attempting to copy it from quay.io"
+        )
+        semantic_version = version.get_semantic_ocs_version_from_config()
+        remote_path = self.get_architecture_path()
+        remote_mcg_cli_basename = os.path.basename(remote_path)
+        local_mcg_cli_dir = os.path.dirname(constants.NOOBAA_OPERATOR_LOCAL_CLI_PATH)
+        if (
+            config.DEPLOYMENT["live_deployment"]
+            and semantic_version >= version.VERSION_4_13
```

Comment:
is version.VERSION 4.13 or 4.12 ?

Reply:
Thanks for pointing it out - we can just choose the 4.12 case in this backport. Resolved in a follow-up commit.